CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
 project(sequence_sketching)
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Any reason for changing this back to 20?

Reply:
changed to `__builtin_popcount` elsewhere to hopefully not need 20.